### DISCUSSION TITLE
Sign Up invalid username/password

### DISCUSSION THREAD

**Discussion by lublak** [cortezaproject/corteza-server#404] [Q&A] [answered]
First start up: Sign Up.
I just startde corteza to give it a try.
![grafik](https://user-images.githubusercontent.com/44057030/186610902-d2cbd641-8a12-4535-93fc-c004dd5b6f6e.png)
![grafik](https://user-images.githubusercontent.com/44057030/186611257-31668fa1-284a-4641-b38d-b1559dea363d.png)

I can't create a user right at the beginning.

**Comment by lublak** [comment, marked as answer]
i delete the complete docker data and created new. Now it works fine.

**Comment by darh** [comment]
What is likely to happen is that you've created the user with the same email and a different password.
